k8s/stg/charts/mi/mi-data-extractor-job/.helmignore

```diff
@@ -1,22 +0,0 @@
-# Patterns to ignore when building packages.
```

Comment:
mi-data-extractor seems to still be in use, can you reach out to Tony Chow and Qiang about it, but remove the changes relating to them from this PR

Reply:
Hello Mohana

We do are using this job, please do not decommission  it!

Kind regards, Qiang